cmake/executable.xcscheme

```diff
@@ -75,12 +75,16 @@
       </AdditionalOptions>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
@kkaefer, this and the cmake/node.xcscheme change look suspicious. Don't we always want to profile the Release configuration, with full optimizations?

Reply:
Yeah, let's drop this change.